pipeline/extract.py

```diff
@@ -0,0 +1,32 @@
+""""Extract script"""
+
+from multiprocessing import Pool
+import requests
+
+
+API_URL = 'https://data-eng-plants-api.herokuapp.com/plants/'
+
+
+def get_data(i: int) -> dict:
+    """
+    Fetches data from the API for a given index.
+    """
+    response = requests.get(f'{API_URL}{i}', timeout=60)
```

Comment:
What happens if this raises a timeout error? 
Look at this article: https://www.geeksforgeeks.org/exception-handling-of-python-requests-module/

Reply:
^^^^ Expanding on the above, I think exception handling would be necessary to stop errors from breaking everything.